android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/view/PdfDisplayDemoView.java

```diff
@@ -0,0 +1,141 @@
+package com.liferay.mobile.formsscreenletdemo.view;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.Matrix;
+import android.graphics.Rect;
+import android.graphics.pdf.PdfRenderer;
+import android.net.Uri;
+import android.os.Build;
+import android.os.ParcelFileDescriptor;
+import android.util.AttributeSet;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import com.liferay.mobile.screens.dlfile.display.pdf.OnSwipeTouchListener;
+import com.liferay.mobile.screens.dlfile.display.pdf.SwipeListener;
+import com.liferay.mobile.screens.util.LiferayLogger;
+import com.liferay.mobile.screens.viewsets.defaultviews.dlfile.display.PdfDisplayView;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * @author Luísa Lima
+ */
+public class PdfDisplayDemoView extends PdfDisplayView {
```

Comment:
I needed to create PdfDisplayDemoView because on Screens the PdfDisplayView change the layout behavior inside a private method. Only change the visibility doesn`t work because this private method cames after the change I did before in the visibility.

Reply:
@camisLuisa The PR was already merged on Screens?

Reply:
It hasn't been merged yet.